src/api/tx_submit.rs

```diff
@@ -9,9 +10,22 @@ pub async fn route(
     // Allow only application/cbor content type
     validate_content_type(&headers, &["application/cbor"])?;
 
-    // Submit transaction
-    let mut node = node.get().await?;
-    let response = node.submit_transaction(body.as_ref()).await?;
+    // XXX: Axum must not abort Ouroboros protocols in the middle, hence a separate Tokio task:
+    let response = tokio::spawn(async move {
```

Comment:
Humm, just asking: Should we not cancel the submission if remote party halts the request?